raitools/raitools/raianalyzer/constants.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+from enum import Enum
+
+
+class ModelTask(str, Enum):
+    """Provide model task constants. Can be 'classification' or 'regression'.
+    """
+
+    Classification = 'classification'
+    Regression = 'regression'
```

Comment:
Probability? For when you have a classification problem but want to work with prediction probabilities for each class?

Reply:
a probability task?  Sorry I'm confused.  Is there a corresponding list of tasks in fairlearn?

Reply:
It's one of the options for Fairlearn, where instead of a predicted class, you have a 'probability of class 1'. Perhaps a PM question?

Reply:
Asked PM to clarify, waiting for PM reply.

Reply:
actually I'm a bit surprised about this issue because in the mock notebook from PM it just says:

```
task_type = 'classification or regression'
```

definitely would be great to get more clarity on this functionality